airbyte-integrations/connectors/source-chargify/integration_tests/sample_config.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Please fill this file according to a valid config.

Reply:
Done!